src/main/java/repository/LottoRepository.java

```diff
@@ -0,0 +1,25 @@
+package repository;
+
+import domain.Lotto;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class LottoRepository {
+    private static List<Lotto> lottos=new ArrayList<>();
+
+    private LottoRepository() {
+    }
+
+    public static LottoRepository create(){
+        return new LottoRepository();
+    }
+
+   public void addLotto(Lotto lotto){
+       lottos.add(lotto);
+   }
```

Comment:
전체적으로 코드 포맷팅이 적용되지 않은 것 같아요. 이런 부분도 코드 컨벤션에 해당하죠. 
코드 컨벤션이 잘 지켜지지 않게 되면, 한 프로젝트를 여러 사람이 수정하는 과정에서 작업하지 않은 부분까지 변경되어 코드 충돌이 발생할 가능성이 높아지더라구요. 전체적으로 개선되면 좋을 것 같아요 🙂

**CTRL + ALT + L** 단축키를 사용하면 자동 포맷팅이 되는데 활용해보셔도 좋을 것 같아요!

```suggestion
    private static List<Lotto> lottos = new ArrayList<>();

    private LottoRepository() {
    }

    public static LottoRepository create() {
        return new LottoRepository();
    }

    public void addLotto(Lotto lotto) {
        lottos.add(lotto);
    }
```

Reply:
google-style 적용했습니다 감사합니다.

Reply:
코드가 훨씬 깔끔해졌네요 👍🏻